Documentation/helm-ceph-cluster.md

```diff
@@ -52,18 +52,34 @@ The following tables lists the configurable parameters of the rook-operator char
 | --------------------- | -------------------------------------------------------------------- | ----------- |
 | `operatorNamespace`   | Namespace of the Rook Operator                                       | `rook-ceph` |
 | `configOverride`      | Cluster ceph.conf override                                           | <empty>     |
+| `ingress`             | Enable an ingress for the ceph-dashboard                             | `{}`        |
 | `toolbox.enabled`     | Enable Ceph debugging pod deployment. See [toolbox](ceph-toolbox.md) | `false`     |
 | `toolbox.tolerations` | Toolbox tolerations                                                  | `[]`        |
 | `toolbox.affinity`    | Toolbox affinity                                                     | `{}`        |
 | `monitoring.enabled`  | Enable Prometheus integration, will also create necessary RBAC rules | `false`     |
 | `cephClusterSpec.*`   | Cluster configuration, see below                                     | See below   |
+| `cephStorageSpec.*`   | Storage configuration, see below                                     | See below   |
 
 
 ### Ceph Cluster Spec
 
 The `CephCluster` CRD takes its spec from `cephClusterSpec.*`. This is not an exhaustive list of parameters.
 For the full list, see the [Cluster CRD](ceph-cluster-crd.md) topic.
 
+### Ceph Storage Spec Spec
```

Comment:
Instead of an overall storage spec, why not include the pool, filesystem, objectstore, ingress, etc. as all top level similar to the cephClusterSpec?
```suggestion
### Ceph Storage Spec
```

Reply:
I've now defined a list of:
`cephBlockPools`
`cephFileSystems`
`cephObjectStores`